Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: rewriting documentation
 Sadly the English in DebugScreen is not quite perfect.
 Hopefully this rewrite captures all the important information
 and will be of benefit to the users.
 We also updated the version number.
Bug: http://rt.cpan.org/Ticket/Display.html?id=69063
Last-Update: 2011-06-24
--- a/ds/lib/CGI/Application/Plugin/DebugScreen.pm
+++ b/ds/lib/CGI/Application/Plugin/DebugScreen.pm
@@ -280,39 +280,29 @@
 
 =head1 VERSION
 
-This documentation refers to CGI::Application::Plugin::DebugScreen version 0.06
+This documentation refers to CGI::Application::Plugin::DebugScreen version 1.00
 
 =head1 SYNOPSIS
 
   use CGI::Application::Plugin::DebugScreen;
 
-Only it.
-If "Internal Server Error" was generated by "run_mode"....
+That is it! The fun starts when a run mode calls C<die>.
 
 =head1 DESCRIPTION
 
-This plug-in add Debug support to CGI::Application.
-This plug-in like Catalyst debug mode.
-
-DebugScreen is done when B<$ENV{CGI_APP_DEBUG}> is set,
- and DebugScreen is not done when not setting it.
-When your code is released, this plug-in need not be removed.
-
-When 'die' is generated by 'run_mode',
- this plug-in outputs the stack trace by error_mode().
-As for this plug-in, error_mode() is overwrited in error callback.
-The error cannot be caught excluding run_mode.
-
-This uses CGI::Application::Plugin::ViewCode
- if a state that CGI::Application::Plugin::ViewCode can be used or used.
-But CGI::Application::Dispatch is used,
- this not uses CGI::Application::Plugin::ViewCode.
-
-When CGI::Application::Plugin::ViewCode can be used,
- Title, Package, File, code and line are links to CGI::Application::Plugin::ViewCode's view_code mode.
-line jumps to the specified line.
-And pod are links to CGI::Application::Plugin::ViewCode's view_pod mode.
-The code of the displayed is links to CGI::Application::Plugin::ViewCode's view_code mode.
+This plug-in adds stack tracing support to CGI::Application similar in style
+to the Catalyst debug mode. In the event of the module calling C<die>,
+a stack trace of the error is displayed. At each level of the stack
+links to the code and documentation are provided.
+
+The stack trace functionality is deactivated unless the B<$ENV{CGI_APP_DEBUG}>
+environment variable is set. Therefore when your code is released,
+this plug-in does not need to be removed.
+
+The links links to code and documentation are implemented using
+L<CGI::Application::Plugin::ViewCode>, which will be loaded automatically
+if available. However these links are not supported in conjunction
+with L<CGI::Application::Dispatch>.
 
 =head1 DEPENDENCIES
 
